common/eth2_network_config/built_in_network_configs/prater/boot_enr.yaml

```diff
@@ -7,4 +7,6 @@
 # Prysm bootnode #1
 - enr:-Ku4QFmUkNp0g9bsLX2PfVeIyT-9WO-PZlrqZBNtEyofOOfLMScDjaTzGxIb1Ns9Wo5Pm_8nlq-SZwcQfTH2cgO-s88Bh2F0dG5ldHOIAAAAAAAAAACEZXRoMpDkvpOTAAAQIP__________gmlkgnY0gmlwhBLf22SJc2VjcDI1NmsxoQLV_jMOIxKbjHFKgrkFvwDvpexo6Nd58TK5k7ss4Vt0IoN1ZHCCG1g
 # Lighthouse bootnode #1
-- enr:-LK4QLINdtobGquK7jukLDAKmsrH2ZuHM4k0TklY5jDTD4ZgfxR9weZmo5Jwu81hlKu3qPAvk24xHGBDjYs4o8f1gZ0Bh2F0dG5ldHOIAAAAAAAAAACEZXRoMpB53wQoAAAQIP__________gmlkgnY0gmlwhDRN_P6Jc2VjcDI1NmsxoQJuNujTgsJUHUgVZML3pzrtgNtYg7rQ4K1tkWERgl0DdoN0Y3CCIyiDdWRwgiMo
+- enr:-Ly4QFPk-cTMxZ3jWTafiNblEZkQIXGF2aVzCIGW0uHp6KaEAvBMoctE8S7YU0qZtuS7By0AA4YMfKoN9ls_GJRccVpFh2F0dG5ldHOI__________-EZXRoMpCC9KcrAgAQIIS2AQAAAAAAgmlkgnY0gmlwhKh3joWJc2VjcDI1NmsxoQKrxz8M1IHwJqRIpDqdVW_U1PeixMW5SfnBD-8idYIQrIhzeW5jbmV0cw-DdGNwgiMog3VkcIIjKA
```

Comment:
I think it probably doesn't make any difference, but these ENRs have Altair fork digests and the actual nodes have forked to Bellatrix

```
Eth2 Field:
	Fork digest: 82f4a72b
	Next fork version: 02001020
	Next fork epoch: 112260
	SSZ Bytes: 82f4a72b0200102084b6010000000000
```

The Bellatrix fork version is `c2ce3aa8`

Reply:
I think this is okay. We will use this node to find other nodes that has bellatrix fork digests, we just won't return nodes with bellatrix fork digests in a discovery requests. The relevant logic is here https://github.com/sigp/lighthouse/blob/df51a73272489fe154bd10995c96199062b6c3f7/beacon_node/lighthouse_network/src/discovery/mod.rs#L760-L767